src/Sulu/Bundle/MediaBundle/Resources/js/components/SingleMediaDropzone/singleMediaDropzone.scss

```diff
@@ -23,6 +24,17 @@ $uploadIndicatorBackgroundColor: rgba($alabaster, .8);
             transform: scale(1);
         }
     }
+
+    &::before{
+        content: '';
+        display: block;
+        padding-top: 100%;
+    }
+
+    @media (max-width: $mediaContainerBreakpoint) {
+        width: 100%;
+        height: auto;
```

Comment:
I think we could set this on the outer selector, instead of setting the width and height, couldn't we? Then we could minimize the code.